File: 프로그래머스/unrated/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    ss = [ord(x)-96 for x in s]         # 아스키 코드 값 변환
    
    for i in range(len(ss)):
        count = index
        while count != 0:

            if ss[i] == 26:
                ss[i] = 0
                
            if(chr(ss[i]+97) not in skip):  # 스킵이 아닌 경우
                count -= 1

            ss[i] += 1
            
    return "".join([chr(x+96) for x in ss])

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.kt
class Solution {
    fun solution(s: String, skip: String, index: Int): String {
        var answer = StringBuilder()

        for (ch in s) {
            var count = index
            var convertCh = ch

            while (count > 0) {
                convertCh += 1

                if (convertCh.code == 'z'.code + 1) {
                    convertCh = 'a'
                }
                
                if (convertCh in skip) {
                    continue
                }
                count--
            }
            answer.append(convertCh)
        }

        return answer.toString()
    }
}